[2025-08-28T18:40:12.702+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-08-28T18:40:12.711+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [queued]>
[2025-08-28T18:40:12.714+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [queued]>
[2025-08-28T18:40:12.714+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-08-28T18:40:12.719+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-08-27 03:00:00+00:00
[2025-08-28T18:40:12.723+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=137) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-28T18:40:12.724+0000] {standard_task_runner.py:72} INFO - Started process 138 to run task
[2025-08-28T18:40:12.724+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'scheduled__2025-08-27T03:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpma17z_aj']
[2025-08-28T18:40:12.725+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask validate_environment
[2025-08-28T18:40:12.747+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [running]> on host b9c1fc824a90
[2025-08-28T18:40:12.779+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='datamasterylab.com' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-08-27T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-27T03:00:00+00:00'
[2025-08-28T18:40:12.780+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-08-28T18:40:12.798+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-08-28T18:40:12.799+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env && \n        echo "Environment is valid!"\n        ']
[2025-08-28T18:40:12.805+0000] {subprocess.py:99} INFO - Output:
[2025-08-28T18:40:12.809+0000] {subprocess.py:106} INFO - Validating environment...
[2025-08-28T18:40:12.809+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-08-28T18:40:12.809+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-08-28T18:40:12.821+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-08-28T18:40:12.821+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=scheduled__2025-08-27T03:00:00+00:00, execution_date=20250827T030000, start_date=20250828T184012, end_date=20250828T184012
[2025-08-28T18:40:12.867+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-08-28T18:40:12.880+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-28T18:40:12.881+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-08-28T18:57:53.941+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-08-28T18:57:53.949+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [queued]>
[2025-08-28T18:57:53.953+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [queued]>
[2025-08-28T18:57:53.953+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-08-28T18:57:53.958+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-08-27 03:00:00+00:00
[2025-08-28T18:57:53.962+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=89) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-28T18:57:53.963+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2025-08-28T18:57:53.963+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'scheduled__2025-08-27T03:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmp_7kjyjvm']
[2025-08-28T18:57:53.964+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask validate_environment
[2025-08-28T18:57:53.983+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment scheduled__2025-08-27T03:00:00+00:00 [running]> on host 1cf203b49fc4
[2025-08-28T18:57:54.013+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nguyentientai223.com' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-08-27T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-27T03:00:00+00:00'
[2025-08-28T18:57:54.013+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-08-28T18:57:54.032+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-08-28T18:57:54.032+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env && \n        echo "Environment is valid!"\n        ']
[2025-08-28T18:57:54.038+0000] {subprocess.py:99} INFO - Output:
[2025-08-28T18:57:54.042+0000] {subprocess.py:106} INFO - Validating environment...
[2025-08-28T18:57:54.042+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-08-28T18:57:54.043+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-08-28T18:57:54.054+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-08-28T18:57:54.054+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=scheduled__2025-08-27T03:00:00+00:00, execution_date=20250827T030000, start_date=20250828T185753, end_date=20250828T185754
[2025-08-28T18:57:54.103+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-08-28T18:57:54.116+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-28T18:57:54.117+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
